views/js/e2e/runner/_setup/setupCommands.js

```diff
@@ -92,5 +95,43 @@ Cypress.Commands.add('setDeliveryForGuests', () => {
 
     // Wait until save happened properly
     // Not ideal but these requests have to be waited in this order upon delivery save
-    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery' ]);
-});
\ No newline at end of file
+    cy.wait(['@editDelivery', '@getData','@editDelivery', '@getData', '@editDelivery', '@getData' ]);
+});
+
+Cypress.Commands.add('startTest', () => {
```

Comment:
I would consider only the startTest as setupCommand, all the rest should be in a separate helper file, navigationCommands for example.

Reply:
I will add navigation comands file.